### PR TITLE
test bG2

### LINKED ISSUE #1: testing readings
**Is your feature request related to a problem? Please describe.**
The[ Design Document on testing](https://github.com/OpenEnergyDashboard/DesignDocs/blob/main/testing/testing.md) specifies readings tests that are desired. Once a test is complete and merged it is labeled as done. We would like the others completed.

**Describe the solution you'd like**
Implement the tests. This is an issue that is generally done by a new developer to get used to working with OED. They normally do 1 or 2 tests for this process and then move on to something else so there are tests for others.

**Additional context**
See information in the design document on doing a test, You must allocate the test as described before beginning work on a test.

### PR DESCRIPTION
**Pull request by nneni**
# Description

I implemented a test addressing the "BG2" quantity group. This test is meant for testing the "Quantity group". This test expanded the program's security by addressing the 7 day bar group. The corresponding data file was uploaded as well, so that the reference made in the testing code would be valid.  

Partially Addresses #1 

## Type of change

(Check the ones that apply by placing an "x" instead of the space in the [ ] so it becomes [x])

- [x] Note merging this changes the database configuration.
- [X] This change requires a documentation update

## Checklist

(Note what you have done by placing an "x" instead of the space in the [ ] so it becomes [x]. It is hoped you do all of them.)

- [X] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [X] I have removed text in ( ) from the issue request
- [X] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.

### CODE REVIEW

File: src/server/test/web/readingsBarGroupQuantity.js [hunk 1]
```diff
@@ -45,7 +45,22 @@ mocha.describe('readings API', () => {
...
+                    const expected = await parseExpectedCsv('src/server/test/web/readingsData/expected_bar_group_ri_15-20_mu_kWh_gu_kWh_st_-inf_et_inf_bd_7.csv')
+
+                    const res = await chai.request(app).get(`/api/unitReadings/bar/groups/${GROUP_ID}`)
+                        .query({ 
+                            timeInterval: ETERNITY.toString(), 
+                            barWidthDays: '7',
+                            graphicUnitId: unitId });
+                    // Check that the API reading is equal to what it is expected to equal
+                    expectReadingToEqualExpected(res, expected, GROUP_ID);
+                });
```
Comment (lawsj) [inline src/server/test/web/readingsBarGroupQuantity.js:63]: This code follow format of  entering the testing , as shown in the previous code lines. Looks good!

**Comment by kate-holdener** [comment]
> # Description
> I implemented a test addressing the "BG2" quantity group. This test expanded the program's security by addressing the 7 day bar group. The corresponding data file was uploaded as well, so that the reference made in the testing code would be valid.
> 
> Partially Addresses #1
> 
> ## Type of change
> (Check the ones that apply by placing an "x" instead of the space in the [ ] so it becomes [x])
> 
> * [x]  Note merging this changes the database configuration.
> * [x]  This change requires a documentation update
> 
> ## Checklist
> (Note what you have done by placing an "x" instead of the space in the [ ] so it becomes [x]. It is hoped you do all of them.)
> 
> * [x]  I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
> * [x]  I have removed text in ( ) from the issue request
> * [x]  You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.
> 
> ## Limitations

Please use:
Fixes #1 
To ensure that the issue is closed when the pull request is merged.

**Comment by kate-holdener** [comment]
> * I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas

This link doesn't work

**Review by Rakesh-Ranga-Buram — COMMENTED** [review]
The data matches with the specifications mentioned in the test documentation and the test written works absolutely fine. Great work!

**Pull request merged**